docs/adr/0003-graphql-layer-between-backend-frontend.md

```diff
@@ -0,0 +1,98 @@
+# GraphQL layer between backend and frontend
+
+Date: 2020-04-30
+
+## Status
+
+**Accepted**
+
+## Related tickets (not available for public)
+
+SIG-2568
+
+## Context
+
+Since our current backend only serves data through REST and there is no possibility to generate
+types and or interfaces, there is no also no automated way to validate data on the frontend
+which is coming from the backend.
+
+Besides of that there are too many inconstencies between different backend calls and the current
```

Comment:
We need to identify the issues that the FE is facing when talking to the API. If we know what these issues are we can determine steps to take and see if we can fix them in the current REST API